### ISSUE TITLE
npm run build fails

### ISSUE THREAD

**Issue by tarun1992** [nielsdejong/neodash#57]
When running the build command, it fails with the below error.
![image](https://user-images.githubusercontent.com/4682201/148782960-6f0d1f7b-93c7-4e17-bc39-b1a727d095fc.png)

In the gitbash, it throws below error - 

![image](https://user-images.githubusercontent.com/4682201/148783164-3705dbe1-a5c0-44ac-a04a-84c2822270b0.png)

Kindly help in making the build as it runs on local server fine.

**Comment by nielsdejong** [comment]
Hi @tarun1992 ,

Likely this happens because you're running the build script from a windows terminal.
The `npm run build` script creates the app and moves some folders around like this:

```
npx webpack --mode production && rm -rf build && mkdir build && cp -r public/* build/ && mkdir build/dist &&  mv dist/ build/
```

In your case, you will probably want to to change it to something like this:
```
npx webpack --mode production && del -rf build && mkdir build && cp -r public/* build/ && mkdir build/dist &&  move dist/ build/
```

(replacing `rm` by `del` and `mv` by `move`)
Could you check if that works?

**Comment by nielsdejong** [comment]
How about this?
`npx webpack --mode production && rmdir build /s /q && mkdir build && cp -r public/* build/ && mkdir build/dist &&  move dist/ build/`
This is pretty difficult for me to debug, as I don't have a Windows system at hand.
I think the issue here is not a `npm`/`npx` problem, but some issues with moving around directories to get the right folder structure. You could also use a different terminal that supports UNIX commands, that might make this easier.

**Comment by tarun1992** [comment]
yes, for now, I manually moved the dist folder inside the build. Somehow, it is not taking these commands.

**Comment by nielsdejong** [comment]
I've just updated the project structure for the 2.0.6 release.
`npm run build` is now a lot simpler:
```
npx webpack --mode production && cp -r public/* dist/
```

or (I guess in windows):
```
npx webpack --mode production && copy -r public/* dist/
```

Does that work for you? If yes, I'll add this solution to the README as it is a lot cleaner.

**Comment by tarun1992** [comment]
Hello, I have checked the new release. It works fine with the new node version but the build command still doesn't work.

![image](https://user-images.githubusercontent.com/4682201/151305891-b6fff00a-6916-423c-a6e5-3a22b7078e59.png)
